### ISSUE TITLE
is there evidence re the cost of TSP versus anything else?

### ISSUE THREAD

**Issue by timm** [ai-se/phaseDelay#21]
one sentence i am writing needs some comment on the overhead of TSP. ideally, the overhead is negligible (after some initial training... how much training? how much cost for an added mentor like the distinguished Dr Nochols? is that mentor needed?)

**Comment by WilliamNichols** [comment]
The overhead is not exactly negligible, but is comparable to running a Scrum team. Recording is negligible, teams usually have a single meeting per week of less than an hour, though some have a daily Scrum type short meeting. The meetings are led by the team leader.
The TSP coach is most active during the project launch and post mortem. A launch seems to take about 2 days for a non-trivial project, with another day or two every 2-3 months. It varies somewhat, but these are typical. Figure 1 day of Coach effort at the end of development cycles (typically 2-3 months) for data analysis, 2 days at end of a longer project. 

Training is a larger investment, with a most basic developer training taking 5 days, the more advanced requiring 10 days of effort.

**Comment by timm** [comment]
Thanks for the above. Could I ask that you rephrase this as something I can add to the paper. e.g. references, learned authoritative statements, the usual drill. just add it as issue text and i'll add it in from there.

**Comment by WilliamNichols** [comment]
I can do that. I will get on that this weekend.

**Comment by WilliamNichols** [comment]
An informed estimate is that tracking time, defects, and tasks requires less then 3% of a developer's time. Weekly team meetings should require at most an hour, which 2.5% of a 40 hour work week. Team launches and replans average about 1 day per month or 5% planning overhead. (Crosstalk,"TSP: Process Costs and Benefits",  James McHale, September, 2002,  A trained coach can support 4 or 6 teams depending upon team experience.
